woocommerce/woocommerce-ios
# [Coupons] Release coupons management feature out of beta

Closes: #9997 


## Description

With this PR we bring the coupons management feature out of beta, so it can always appear in the Menu.

## Testing instructions


1. Go to Menu. Check that the Coupons feature is available there.
2. Go to Settings. 
3. Go to Experimental Features.
4. Check that Coupons is no longer available there.

## Screenshots


<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/da65cc7c-cc74-435e-a384-f173b8bbc229" width="375">
<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/926df72e-dff6-498b-a905-ba8f9675254b" width="375">

---
